SUMMARY.md

```diff
@@ -6,46 +6,45 @@
 
-- [Top Earlgrey](./hw/top_earlgrey/README.md)
-  - [Datasheet](./hw/top_earlgrey/doc/datasheet.md)
-  - [Design](./hw/top_earlgrey/doc/design/README.md)
-  - [Design Verification](./hw/top_earlgrey/dv/README.md)
-    - [Chip Testplan](./hw/top_earlgrey/data/chip_testplan.hjson)
-  - [Analog Sensor Top](./hw/top_earlgrey/ip/ast/README.md)
-  - [Alert Handler](./hw/top_earlgrey/ip_autogen/alert_handler/README.md)
-    - [Theory of Operation](./hw/top_earlgrey/ip_autogen/alert_handler/doc/theory_of_operation.md)
-    - [Design Verification](./hw/top_earlgrey/ip_autogen/alert_handler/dv/README.md)
-      - [Testplan](./hw/top_earlgrey/ip_autogen/alert_handler/data/alert_handler_testplan.hjson)
-    - [Programmer's Guide](./hw/top_earlgrey/ip_autogen/alert_handler/doc/programmers_guide.md)
-    - [Interface and Registers](./hw/top_earlgrey/ip_autogen/alert_handler/data/alert_handler.hjson)
-    - [Device Interface Functions](./sw/device/lib/dif/dif_alert_handler.h)
-    - [Checklist](./hw/top_earlgrey/ip_autogen/alert_handler/doc/checklist.md)
-  - [Interrupt Controller](./hw/top_earlgrey/ip_autogen/rv_plic/README.md)
-    - [Theory of Operation](./hw/top_earlgrey/ip_autogen/rv_plic/doc/theory_of_operation.md)
-    - [Design Verification](./hw/top_earlgrey/ip_autogen/rv_plic/doc/dv/README.md)
-      - [Testplan](./hw/top_earlgrey/ip_autogen/rv_plic/data/rv_plic_fpv_testplan.hjson)
-    - [Programmer's Guide](./hw/top_earlgrey/ip_autogen/rv_plic/doc/programmers_guide.md)
-    - [Interface and Registers](./hw/top_earlgrey/ip_autogen/rv_plic/data/rv_plic.hjson)
-    - [Device Interface Functions](./sw/device/lib/dif/dif_rv_plic.h)
-    - [Checklist](./hw/top_earlgrey/ip_autogen/rv_plic/doc/checklist.md)
-  - [Sensor Control](./hw/top_earlgrey/ip/sensor_ctrl/README.md)
-    - [Theory of Operation](./hw/top_earlgrey/ip/sensor_ctrl/doc/theory_of_operation.md)
-    - [Programmer's Guide](./hw/top_earlgrey/ip/sensor_ctrl/doc/programmers_guide.md)
-    - [Interface and Registers](./hw/top_earlgrey/ip/sensor_ctrl/data/sensor_ctrl.hjson)
-    - [Device Interface Functions](./sw/device/lib/dif/dif_sensor_ctrl.h)
-    - [Checklist](./hw/top_earlgrey/ip/sensor_ctrl/doc/checklist.md)
-  - [TL-UL Checklist](./hw/top_earlgrey/ip/xbar/doc/checklist.md)
-  - [Pinmux Targets](./hw/top_earlgrey/ip/pinmux/doc/autogen/targets.md)
-    - [ASIC Target Pinout and Pinmux Connectivity](./hw/top_earlgrey/ip/pinmux/doc/autogen/pinout_asic.md)
-    - [CW310 Target Pinout and Pinmux Connectivity](./hw/top_earlgrey/ip/pinmux/doc/autogen/pinout_cw310.md)
+- [Tops](./doc/tops.md)
+  - [Earl Grey](./hw/top_earlgrey/README.md)
+    - [Datasheet](./hw/top_earlgrey/doc/datasheet.md)
+    - [Design Specification](./hw/top_earlgrey/doc/design/README.md)
+    - [Design Verification](./hw/top_earlgrey/dv/README.md)
+      - [Chip Testplan](./hw/top_earlgrey/data/chip_testplan.hjson)
+    - [Top Generation](./hw/top_earlgrey/doc/design/top_generation.md)
+    - [Earl Grey HWIP]()
```

Comment:
Using the 'Hardware Blocks' wording as chapters in the navbar seems to be popular, so I will switch to that.

At some point soon it is probably worth agreeing on common terminology for OTI and then add a glossary to document it. 'Generic Hardware Blocks' or 'HWIP Library'? It OTI a system or a subsystem? When to use Hardware Blocks or HWIP? It would certainly make our documentation more cohesive if we could standardize on terms.

Reply:
Sounds good, thanks for changing it.

What do you mean by OTI? OpenTitan IP?

I agree that standardization would be good here.
Do you mind filing an issue with the ambiguities you've come across?
That way we can collect and discuss these in a future silicon WG meeting.

Reply:
Sorry, OTI was an acryonym for opentitan integrated that some people were throwing around here.

I think I will put together an RFC for adopting standard Terminology to gather discussion and possibly choose terms to adopt more-formally.